media/starboard/starboard_renderer.h

```diff
@@ -56,35 +56,14 @@ class MEDIA_EXPORT StarboardRenderer final : public Renderer,
                   PipelineStatusCallback init_cb) final;
   void SetCdm(CdmContext* cdm_context, CdmAttachedCB cdm_attached_cb) final;
   void SetLatencyHint(absl::optional<base::TimeDelta> latency_hint) final {
-    // TODO(b/375271848): Address NOTIMPLEMENTED().
-    NOTIMPLEMENTED();
-  }
-  void SetPreservesPitch(bool preserves_pitch) final {
-    // TODO(b/375271848): Address NOTIMPLEMENTED().
-    NOTIMPLEMENTED();
-  }
-  void SetWasPlayedWithUserActivation(
-      bool was_played_with_user_activation) final {
-    // TODO(b/375271848): Address NOTIMPLEMENTED().
```

Comment:
Consider adding a LOG(INFO) here for any functions we decided not to implement with the value of the parameters, so we'd know that they are called.

Reply:
Addressed this in https://github.com/youtube/cobalt/pull/4558.